src/lib/prompts.ts

```diff
@@ -24,6 +24,13 @@ import commitMessage, {
 import commitlint from './commitlint';
 import localize, { locale } from './localize';
 
+type ConventionalCommitsTypes = {
```

Comment:
Add type definition, and remove `ts-ignore`

Reply:
Thanks! Should I update the definition into my package?

Reply:
@yi-Xu-0100 Your package is fine. Types definitions in your package are more strict than this extension. 😀

I think we should work on the types definitions within the extension. But I have no better ideas right now.